cylc/flow.cylc

```diff
@@ -0,0 +1,161 @@
+#!jinja2
+
+[scheduler]
+    allow implicit tasks = True
+
+[task parameters]
+    satellite = {{ SATELLITES | join(", ")}}
+
+[scheduling]
+    initial cycle point = {{ START }}
+    final cycle point = {{ END }}
+    [[graph]]
+        R1 = """
+            INIT 
+            INIT_soit => soit
+        """
+        P1D = """
+            INIT[^]:succeed-all & soit[^] => preprocess<satellite>
+            INIT[^]:succeed-all => FETCHDATA_truecolor<satellite> & FETCHDATA_falsecolor<satellite> & FETCHDATA_landmask & FETCHDATA_cloud<satellite> => preprocess<satellite> => cycle_end
+            FETCHDATA_soit_daily<satellite> => cycle_end
+            cycle_end[-P1D] => cycle_end
+            
+            preprocess<satellite> & FETCHDATA_soit_daily<satellite> => exportH5
+        """
+        R1/P0Y = cycle_end => tracking 
+
+
+[runtime]
+    [[root]]
+        
+        work sub-directory = .
+        
+        [[[environment]]]
+            # Update these variables with your run parameters
```

Comment:
This part is a little confusing on what happens here. Does the user need to interact with this `flow.cylc` now or does cylc inherently know to look for `rose-suite.conf`? Along those lines, does using rose add another dependency that we could avoid?

Reply:
All the editable params will be in the rose-suite.conf